source_gen/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.1.0-dev
```

Comment:
I would probably drop the `-dev` and we can go ahead and publish (pubspec also). I don't think we have any other pending changes in this package.